script-library/container-features/src/features.json

```diff
@@ -1065,12 +1065,61 @@
 				"nonFreePackages": {
 					"type":"boolean",
 					"default": true,
-					"description": "Add packages from non-free Debain repository?"
+					"description": "Add packages from non-free Debian repository?"
 				}
 			},
 			"buildArg": "_VSC_INSTALL_COMMON",
-			"include": []
+			"include": [
+				"puppet",
+				"chef"
+			]
+		},
+		{
+			"id": "dotnet",
+			"name": "Dotnet CLI",
+			"documentationURL": "https://github.com/microsoft/vscode-dev-containers/blob/main/script-library/docs/dotnet.md",
+			"options": {
+				"version": {
+					"type": "string",
+					"proposals": ["latest","6.0","5.0","3.1"],
+					"default": "latest",
+					"description": "Select or enter a dotnet CLI version. (Available versions may vary by Linux distribution.)"
+				},
+				"runtimeOnly": {
+					"type":"boolean",
+					"default": false,
+					"description": "Install just the dotnet runtime if true, and sdk if false."
+				}
+			},
+			"buildArg": "_VSC_INSTALL_DOTNET",
+			"extensions": [
+				"ms-dotnettools.csharp"
+			],
+			"include": [
+				"cpp",
+				"kubernetes-helm",
+				"kubernetes-helm-minikube",
+				"docker-from-docker",
+				"docker-in-docker",
+				"powershell",
+				"java",
+				"go",
+				"php",
+				"ruby",
+				"rust",
+				"typescript-node",
+				"javascript-node",
+				"python-3",
+				"python-3-anaconda",
+				"python-3-miniconda",
+				"ubuntu",
+				"debian",
+				"javascript-node-mongo",
+				"javascript-node-postgres",
+				"php-mariadb",
+				"ruby-rails-postgres",
+				"python-3-postgres"
```

Comment:
are all these new includes needed?

Reply:
Yes, they are all base containers that this feature can be run with and if they are not included they will not be available via the command palette.